cortex-js/src/utils/logs.ts

```diff
@@ -0,0 +1,32 @@
+import { createReadStream } from 'fs';
+import { join } from 'path';
+import { createInterface } from 'readline';
+
+/**
+ * Print the last N lines of a file that contain the word 'ERROR'
+ * @param filename
+ * @param numLines
+ */
+export async function printLastErrorLines(
+  dataFolderPath: string,
+  numLines: number = 5,
```

Comment:
stupid qn from me - any reason why only 5 lines (will they see a truncated error message?)

Reply:
non blocking curious qn, otherwise LGTM!

Reply:
> stupid qn from me - any reason why only 5 lines (will they see a truncated error message?)

Yeah, that's a rough estimate for common cases. Since we output everything into the log file across sessions, we should output only several lines (not just errors), similar to other applications. Otherwise, there would be hundreds of lines of cortex-cpp printed out. The applications are not running in attached mode; in attached mode, users would see the full session logs.